manifests/backports.pp

```diff
@@ -1,3 +1,4 @@
+# Defining backports for the APT class
```

Comment:
i hate to nit but i'm going to. apt isn't an acronym so it doesn't need to be capitalized :)